### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by lukasmatta** [AbsaOSS/cps-shared-ui#426]
There is a room for improvement when it comes to accessibility. Proper accessibility testing should be performed using both manual testing and automatic tools such as [Axe](https://www.deque.com/axe/). Here is the list of issues I found during brief manual testing:

- some components use `px` unit for font size instead of `rem`
- complex components such as `ButtonToggle`, or `Tabs` cannot be controlled using keyboard
- some texts do not satisfy the [minimum contrast requirement](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html), e.g. [right info text in the autocomplete component](https://github.com/AbsaOSS/cps-shared-ui/blob/afb745cf9717fadb90b08f09c03c3dc1cce55423/projects/cps-ui-kit/src/lib/components/cps-autocomplete/cps-autocomplete.component.scss#L335), [input hint text](https://github.com/AbsaOSS/cps-shared-ui/blob/afb745cf9717fadb90b08f09c03c3dc1cce55423/projects/cps-ui-kit/src/lib/components/cps-input/cps-input.component.scss#L215)
- `label` elements are not referencing related input elements using either `for` attribute or being wrapped in it

## Tasks
- [ ] Perform accessibility test using both manual testing and automatic tool and create a report
- [ ] Fix accessibility issues
- [ ] Perform a control test and compare its results with the previous one
